scripts/xref_check.eterm

```diff
@@ -0,0 +1,536 @@
+[ %% Check undefined_function_calls
+  #{ name => undefined_function_calls
+   , analysis => undefined_function_calls
+   , excl_apps =>
+         [ observer
+         , redbug
+         ]
+   , excl_mods =>
+         [ hipe_unified_loader
+         , systools_make
+         , systools
+         , systools_relup
+         , basho_bench_driver_erldis
+         , release_handler
+         , cuttlefish_rebar_plugin
+         ]
+   , filters =>
+         [{{coap_client,channel_apply,3},{coap_dtls_socket,close,1}},
+          {{coap_client,channel_apply,3},{coap_dtls_socket,connect,2}},
+          {{coap_client,channel_apply,3},{coap_udp_socket,close,1}},
+          {{coap_client,channel_apply,3},{coap_udp_socket,get_channel,2}},
+          {{coap_client,channel_apply,3},{coap_udp_socket,start_link,0}},
+          {{coap_client,test,0},{eunit,test,1}},
```

Comment:
Q: are eunit/test functions in the release then?  :thinking:

Reply:
I guess so if xref say so.

Reply:
:hear_no_evil: